.github/workflows/run_system_tests.yml

```diff
@@ -22,6 +22,8 @@ jobs:
     steps:
       - name: Check out repo
         uses: actions/checkout@v4
+      - name: Rename .env.simulation to .env
+        run: mv examples/.env.simulation .env
```

Comment:
Moving this file dirties the Git checkout. You should copy it instead.

Reply:
Updated `mv` to `cp` to copy files instead of moving.